jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/MultiObjTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jmxmetrics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.utility.MountableFile;
+
+class MultiObjTest extends AbstractIntegrationTest {
```

Comment:
Since this looks like a new feature of the helper rather than a new backend integration I think it's better not to add yet another integration test to make sure they don't get too slow.

How about adding unit test to instrumenthelpertest and modifying the existing Cassandra test to exercise this?

Reply:
[here](https://github.com/open-telemetry/opentelemetry-java-contrib/pull/43#discussion_r690663935) I had originally setup a modified version of the cassandra test and was told to revert it and add an additional test.

Reply:
My second request to keep the single mbean integration test as is and add a new one with multiples was to preserve the datapoint validation that was being removed in favor of a simple size check that was apparently flakey.

Reply:
I'm personally a fan of integration tests but can understand not want to balloon CI and dev times.  If there is to be a unit test instead of the current approach I think checking the functionality of the InstrumentHelper with multi-mbean MBeanHelper would be a good way forward.

Reply:
So, can this be merged? or are the changes required to be unit tests? @rmfitzpatrick

Reply:
@anuraaga are you ok with the changes as is?  Improving coverage could be a future item.

Reply:
I'd like to have the test change made yes - it's why I left a comment :P The integration tests are already up to 5 min compared to 2 for unit tests so I don't want to continue trending up.